src/java.base/share/classes/javax/crypto/CryptoPermission.java

```diff
@@ -442,20 +433,13 @@ private boolean impliesParameterSpec(boolean checkParam,
 
             // For classes we don't know, the following
             // may be the best try.
-            if (this.algParamSpec.equals(algParamSpec)) {
-                return true;
-            }
-            return false;
-        } else if (this.checkParam) {
-            return false;
-        } else {
-            return true;
-        }
+            return this.algParamSpec.equals(algParamSpec);
+        } else return !this.checkParam;
```

Comment:
Please use the standard coding format.

`
} else { 
    return !this.checkParam;
}
`

Reply:
Using the first format.